.github/workflows/php.yml

```diff
@@ -104,26 +104,22 @@ jobs:
     - name: PHPUnit Check
       run: composer test
 
-    - name: Create Release
-      id: create_release
-      uses: actions/create-release@v1
-      if: startsWith(github.ref, 'refs/tags/')
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    - name: Pre-release
+      uses: marvinpinto/action-automatic-releases@latest
```

Comment:
Why the change of GitHub Action to use here?

Reply:
`actions/create-release` has some issues that tries to create a tag even though a tag already exists, which caused this build to fail: https://github.com/SwedbankPay/swedbank-pay-magento2-core/runs/1861672894?check_suite_focus=true

I noticed you also replaced the package in Checkout module: SwedbankPay/swedbank-pay-magento2-checkout@5a0f542

Reply:
It would be great if the description of the pull request could contain these sorts of motivations and explanations. 👍🏼